.github/workflows/new-runtime-upgrade.yml

```diff
@@ -0,0 +1,141 @@
+name: "WASM runtime upgrade"
+
+on:
+  workflow_dispatch:
+    inputs:
+      name:
+        type: choice
+        required: true
+        description: 'Network environment'
+        options:
+          - dali
+          - composable
+          - picasso
+      release_selection:
+        type: string
+        required: true
+        description: 'Release version. Example: v2.2.0'
+
+jobs:
```

Comment:
if fills if condirion should define set of vars, and then all blocks run without much change. 
so instead of copy paste 3 times you paramterize.
in future that may be template of set of jobs tasks chosen per runtime upgrade.
so not sure if current copy paste is good now.

Reply:
Its a temporary logic.